srfi-tools/make-chibi-image.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
```

Comment:
`cd "$(dirname "$0")"` to change to the directory where the script it. Once you're in that directory, you can `cd ..` and so on.